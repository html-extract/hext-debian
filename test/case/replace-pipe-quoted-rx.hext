<a @text:trim:replace('content'i, "\"strange\"/days/"):replace("\"strange\"/days/"i, "content"):replace("content", 'potato'):text />
