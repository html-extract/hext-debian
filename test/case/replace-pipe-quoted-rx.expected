{"text":"some more potato"}
{"text":"This is some potato"}
